feature/security/gnsi/acctz/AccountingAuthenErrorMulti/README.md

```diff
@@ -0,0 +1,53 @@
+# gNSI.acctz.v1 (Accounting) Test Accounting Authentication Error - Multi-transaction
+
+Test Accounting for authentication errors of multi-transaction logins
```

Comment:
you might say how 'errors' is different from the next test readme 'failure'

Reply:
I think I have addressed this; but possibly not as explicitly as you might like.  You can judge.

Reply:
is good! :)